src/content/index.ts

```
@@ -70,16 +63,13 @@ chrome.runtime.onMessage.addListener(async (message) => {
       currentCreatorIndex = 0
       break
 
-    case ActionType.TOGGLE_SIDEBAR:
+    case ActionType.TOGGLE_SIDE_PANEL:
       const aduSidePanelDiv = document.getElementById('adu-sidepanel-container')
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)